Articles/WhatIs/WhatIsPeerCodeReview.md

```diff
@@ -0,0 +1,61 @@
+### What is Peer Code Review?
+#### Contributed by  [Roscoe A. Bartlett](https://github.com/bartlett)
+#### Publication date: ???
+
+<!--- deck start --->
+
+**Peer Code Review** is a process in which one or more developers inspect another developer's software code to assess the user interfaces, software implementation, documentation, and other software-related artifacts.
+In addition, these reviews can help to catch potential defects, security vulnerabilities, performance bottlenecks, sustainability impediments, and many other software-related issues.
+
+<!--- deck end --->
+
+<!--- body start --->
+
+Peer code reviews are an essential part of the software development process where, along with software testing, provide the foundation for software quality assurance.
```

Comment:
The `...where...provide...` wording is awkard. Seems like a word is missing.

Reply:
Agreed.  Will reword.

Reply:
Should be fixed in new commit just pushed [`d9daf6f` (#1678)](https://github.com/betterscientificsoftware/bssw.io/pull/1678/commits/d9daf6fb9c82a79d91730811cbce719863bd2954).  Let me know what you think.